packages/lib/.husky/pre-commit

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+#. "$(dirname "$0")/_/husky.sh"
+
+# helper if you are running repo inside a container
+if which vagrant; then
+  vagrant ssh -- -t <<HEREDOC
+  cd workspace/adyen-web/packages/lib;
+  echo "head -n -1 ~/.bashrc > temp.txt ; mv temp.txt ~/.bashrc && npx lint-staged;" >> ~/.bashrc;
+  /bin/bash -i
+HEREDOC
+else
+  npx lint-staged
+fi
```

Comment:
Any reason why this is placed inside the 'lib' package? Why not put it and the husky dependency in the root level? It could potentially be used in other packages too

Reply:
The main reason is that we only had prettier on the lib/ project.